packages/surveycto/src/Utils.js

```diff
@@ -50,3 +51,43 @@ export const requestHelper = (state, path, params, callback = s => s) => {
       throw err;
     });
 };
+
+export const dateRegex = /^(\w{3} \d{2}, \d{4} \d{2}:\d{2}(:\d{2})? (PM|AM))$/;
+
+/**
+ * This function will attempt to convert any date representation into
+ * a surveyCTO `MMM dd, yyy h:mm:ss a` string.
+ * Strings already in this format will be ignored, other strings will be parsed
+ * by the Date constructor.
+ * Number values should be epoch or unix timestamps and will be converted to strings
+ * @param {*} date a date in a string, number or Date format
+ * @returns
+ */
+export const convertDate = date => {
+  // If it's already in the right format, return it
+  if (typeof date === 'string' && dateRegex.test(date)) {
+    return date;
+  }
+
+  // Otherwise parse the input into a new date object
+  let dateObj = date;
+  if (typeof date === 'string' || typeof date === 'number') {
+    if (/^\d{10}$/.test(date)) {
+      // If the incoming date is a unit timestamp, just return it
+      dateObj = new Date(date * 1000);
+    } else {
+      dateObj = new Date(date);
+    }
+  }
+
+  // And return in the correct formatting (utc time)
+  const formatted = formatInTimeZone(
+    dateObj.toISOString(),
+    'UTC',
+    'MMM dd, yyy h:mm:ss a'
+  );
+
+  console.log(`Converted timestamp "${date}" to "${formatted}" (UTC)`);
```

Comment:
I added this as an afterthought because it seems quite important :thinking: 

On the other hand it's noisy and maybe a bit defensive. Opinions welcome!